iris-mpc-cpu/src/hawkers/galois_store.rs

```diff
@@ -250,6 +250,9 @@ impl VectorStore for LocalNetAby3NgStoreProtocol {
         query: &Self::QueryRef,
         vectors: &[Self::VectorRef],
     ) -> Vec<Self::DistanceRef> {
+        if vectors.is_empty() {
+            return vec![];
+        }
```

Comment:
(See my [comment](https://github.com/worldcoin/iris-mpc/pull/797#discussion_r1884286781) in #797.)